libraries/passive_sensitive.rb

```diff
@@ -0,0 +1,16 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: passive_sensitive.rb
+
+require 'chef/resource'
+
+# rubocop:disable Documentation
+class Chef
+  # Making the sensitive attribute passive for older chef versions
+  class Resource
+    def sensitive(_ = nil)
+    end
+  end
+end unless Chef::Resource.method_defined? :sensitive
+# rubocop:enable Documentation
```

Comment:
I don't think it would be prudent to duplicate the functionality of the sensitive flag for older versions of chef. There was a good amount of code that changed and I feel like maintenance would be a pain.